config/locales/doc_auth/en.yml

```diff
@@ -67,13 +68,19 @@ en:
         your identity.
       document_capture_upload_image: We only use your ID to verify your identity,
         and we will not save any images.
+      id_worn_html: "<strong>Is your ID worn or damaged?</strong> Use another state-issued
+        ID if you have one."
       interstitial_eta: This might take up to a minute. We’ll load the next step automatically
         when it’s done.
       interstitial_thanks: Thanks for your patience!
       link_sent:
       - Please check your phone and follow instructions to take a photo of your state
         issued ID.
       - When you are done click continue here to finish verifying your identity.
+      no_other_id_help_bold: "<lg-strong>If you do not have another state-issued ID</lg-strong>,
+        <lg-get-help-link>get help at %{sp_name}.</lg-get-help-link>"
+      no_other_id_help_bold_html: "<strong>If you do not have another state-issued
+        ID</strong>, <a href=%{failure_to_proof_url}>get help at %{sp_name}.</a>"
```

Comment:
I'm feeling that there's a bit of inconsistency with how I used strings here:

- `id_worn_html` is reused between both the Rails-rendered `upload.html.erb` and the React application, where the React application will render `<strong>` as `<strong>`, but in the case of `no_other_id_help_bold`, the string uses the `<lg-strong>` custom tag.
   - Previous conversation: https://github.com/18F/identity-idp/pull/3997#discussion_r463255571
- It wasn't clear to me previously that `_html` suffix is meaningfully understood by Rails to avoid escaping. `no_other_id_help_bold` is a strange case in how it includes HTML, and while we'd not intend for it to be used in Rails-rendered views, its HTML would be escaped if it ever were to be.

I'm contemplating if it might make sense to introduce a new convention to more strongly signal strings which include "HTML" that we're expecting only to be used in a JavaScript context, like a `_js` suffix. And if we were to go this route, if it would also make sense to split `id_worn_html` to `id_worn_js` with `<lg-strong>` tags but otherwise identical text.

Reply:
Since the text of these the exact same (and carry the same meaning AFAIK) I wish that we could use the same key both in rails and in React.

Would it possible to update our HTML tag parsing code to swallow extra attributes after a tag name, ex if we added the `href`?

```
<a href=%{failure_to_proof_url}>
```
would just be the same as
```
<a>
```

Reply:
Yeah, the current JavaScript HTML formatting is rather naive, but I could see it being enhanced to support something like what you're describing. I'll take a look at it.

Regarding `lg-`, I think it also made a bit more sense with #3997 where underlining text in the way we were intending to do wasn't as well represented semantically by an equivalent HTML element (where [`<u>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u) wasn't appropriate).

Reply:
Added support for matching HTML attributes to `formatHTML` in 58c1479.

Removed and substituted the extra `no_other_id_help_bold` with `no_other_id_help_bold_html` in 3dd4c7a.